lib/controllers/AssetController.ts

```diff
@@ -29,6 +29,11 @@ export class AssetController extends RelationalController {
       description: 'Metadata which are specified in AssetCategory as mandatory must be present',
       message: 'metadata %s is mandatory for the asset',
     });
+    global.app.errors.register('device-manager', 'assetController', 'EnumMetadata', {
```

Comment:
Error codes should be in camelCase (sorry this isn't documented nor tested in Kuzzle..)
```suggestion
    global.app.errors.register('device-manager', 'asset_controller', 'enum_metadata', {
```

Reply:
Done (and I correct all the others error code that was present before this PR)